tests/readPreference/readpreference-set_state_error-001.phpt

```diff
@@ -7,42 +7,46 @@ require_once __DIR__ . '/../utils/basic.inc';
 
 echo throws(function() {
     MongoDB\Driver\ReadPreference::__set_state(['mode' => 'furthest']);
-}, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
+}, MongoDB\Driver\Exception\InvalidArgumentException::class), "\n";
```

Comment:
Did a search/replace while I was in this file to ensure it was consistent with itself.